llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -257,7 +257,12 @@ class YkControlPoint : public ModulePass {
     Builder.SetInsertPoint(BB);
     Builder.CreateCondBr(NewCtrlPointCallInst, ExitBB, ContBB);
 
-    // Generate new control point logic.
+    // Our pass runs after LLVM normally does its verify pass. In debug builds
```

Comment:
Dumb question: this doesn't seem to be guardade for debug builds?

Reply:
You passed the test! Example correct answer pushed in 8b159bdfed1f ;)

Reply:
How clever you are to try and trick me :p